etc/vagrant/install.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+# Script to set up a Django project on Vagrant.
+PROJECT_NAME=$1
+
+DB_NAME=$PROJECT_NAME
+DB_USERNAME=$PROJECT_NAME
+DB_PASSWD=$PROJECT_NAME
+VIRTUALENV_NAME=$PROJECT_NAME
+
+PROJECT_DIR=/home/vagrant/$PROJECT_NAME
+VIRTUALENV_DIR=/home/vagrant/.virtualenvs/$PROJECT_NAME
+
+
+# Install essential packages from Apt
+apt-get update -y
+
+# Python dev packages
+#apt-get install -y build-essential python python3-dev npm
+apt-get build-dep -y python-imaging
+apt-get install -y python3-pip \
+        libjpeg8 \
+        libjpeg62-dev \
+        libfreetype6 \
+        libfreetype6-dev \
+        libpq-dev \
+        wget
+
+# Git
+apt-get install -y git
+
+# Postgresql
+if ! command -v psql; then
+    apt-get install -y postgresql libpq-dev
+fi
+
+# Create vagrant pgsql superuser
+su - postgres -c "createuser -s vagrant"
+
+
+if ! command -v pip; then
```

Comment:
is it possible, that `pip` will not be available if we have `apt-get install python3-pip` above (remember, that we support python3 only here)